### DISCUSSION TITLE
Auto close and rename HTML tags

### DISCUSSION THREAD

**Discussion by affanmustafa** [zed-industries/zed#7883] [Feature Requests]
I have noticed that Zed does not auto close the HTML tags nor does it rename the closing tag if you rename the opening one. 

Could this perhaps be added in a future updates list?

Attaching a video below

https://github.com/zed-industries/zed/assets/69677074/8666d1b3-b755-4796-bd13-ab787f8375c1

**Comment by SomeoneToIgnore** [comment]
It could, there are issues for both features: https://github.com/zed-industries/zed/issues/4535 , https://github.com/zed-industries/zed/issues/5267
